patches/@types+marked+0.6.5.patch

```diff
@@ -1,12 +0,0 @@
-diff --git a/node_modules/@types/marked/index.d.ts b/node_modules/@types/marked/index.d.ts
```

Comment:
Since this is no longer needed, can we remove the part of the `check-types` and `postinstall` scripts in package.json that call `patch-package`?

Reply:
Done!